assets/src/features/room-page/components/Sidebar.tsx

```diff
@@ -0,0 +1,43 @@
+import clsx from "clsx";
+import React, { useState } from "react";
+import { LocalPeer, RemotePeer } from "../../../pages/room/hooks/usePeerState";
+import Chat from "./Chat";
+import PeopleComponent from "./PeopleComponent";
+import SidebarTab, { Tab } from "./SidebarTab";
+
+type SidebarProps = {
+  peers: RemotePeer[];
+  localPeer?: LocalPeer;
+};
+
+const Sidebar: React.FC<SidebarProps> = ({ peers, localPeer }) => {
+  const [tab, setTab] = useState<Tab>("people");
+
+  return (
+    <div
+      className={clsx(
+        "border border-brand-dark-blue-300 bg-brand-white",
+        "rounded-xl",
+        "grid-wrapper hidden w-[300px] min-w-[300px] flex-col md:flex",
```

Comment:
I don't see purpose in using both `w-[300px]` and `min-w-[300px]`. If I remember correctly, we agreed on fixed 300px on this